content/doc/developer/forms/form-validation.adoc

```diff
@@ -45,6 +45,14 @@ public FormValidation doCheckThreads(@QueryParameter("value") final String meani
 
 This naming convention is how Jenkins wires up your form validation method. The return type also must be FormValidation, which offers various static factory methods to create an instance. Form validation can be of 3 different kinds: `ok`, `warning` or `error`. Validation message can either be represented in plaintext or use HTML markup. Several validations of different kinds can be reported wrapping then into one using aggregate factory method.
 
+=== To set up form validation within a Jelly template
+
+To configure form validation within a Jelly template to call doCheckXxx methods defined in a particular class in Jenkins, you'll indeed utilize the checkUrl and checkDependsOn attributes. Here's a breakdown of how these attributes are used:
+
+The checkUrl Attribute specifies the URL that Jenkins will request to perform form validation. Typically, this URL points to the Jenkins server, where it handles the validation request. The value entered in the input field will be checked via AJAX against this URL, and any errors encountered will be rendered under the text field. This URL is associated with the method responsible for performing the validation (doCheckXxx method) within your Descriptor class.
```

Comment:
```suggestion
The `checkUrl` attribute specifies the URL that Jenkins will request to perform form validation.
Typically, this URL points to the Jenkins server where it handles the validation request.
The value entered in the input field will be checked via AJAX against this URL and any errors encountered will be rendered under the text field.
This URL is associated with the method responsible for performing the validation (`doCheckXxx` method) within your Descriptor class.
```
Suggestion to use code span for user entered attributes, and change the case on attribute to match the other usage of it.
Suggestion to make each full sentence (with full stop) a single line (number).